packages/customize-widgets/src/components/sidebar-block-editor/index.js

```diff
@@ -28,33 +35,64 @@ export default function SidebarBlockEditor( { sidebar } ) {
 		modal: false,
 		animated: 150,
 	} );
+	const [ isInspectorOpened, setIsInspectorOpened ] = useState( false );
 	const settings = useMemo(
 		() => ( {
 			__experimentalSetIsInserterOpened: inserter.setVisible,
 		} ),
 		[ inserter.setVisible ]
 	);
+	const collapse = () => setIsInspectorOpened( false );
 
 	return (
-		<SlotFillProvider>
-			<DropZoneProvider>
-				<BlockEditorProvider
-					value={ blocks }
-					onInput={ onInput }
-					onChange={ onChange }
-					settings={ settings }
-				>
-					<Header inserter={ inserter } />
-
-					<BlockSelectionClearer>
-						<WritingFlow>
-							<ObserveTyping>
-								<BlockList />
-							</ObserveTyping>
-						</WritingFlow>
-					</BlockSelectionClearer>
-				</BlockEditorProvider>
-			</DropZoneProvider>
-		</SlotFillProvider>
+		<>
+			<BlockEditorKeyboardShortcuts.Register />
+			<SlotFillProvider>
+				<DropZoneProvider>
+					<div hidden={ isInspectorOpened }>
```

Comment:
This hides the block editor _before_ the left↔right animation begins which causes a flicker in the animation. We should hide it _after_ the animation so that things look smooth.